torch/_torch_docs.py

```diff
@@ -3113,6 +3113,56 @@ def merge_dicts(*dicts):
 
 """)
 
+add_docstr(torch.inner, r"""
+inner(input, other, *, out=None) -> Tensor
+
+Computes the vector inner product for 1D tensors. In higher dimensions, computes the sum
```

Comment:
Going to delay reviewing docs for a moment until we resolve the equivalency to `torch.tensordot(a_t, b_t, dims=((-1,), (-1,)))`, since if that's the case the docs become easier to write.

Reply:
Replied above.